README.md

```diff
@@ -1,2 +1,46 @@
-# Portafolio
-This project shows all my experience as a software Developer
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+It is my portfolio, experience has a software developer
+
+> Description the project.
+- **This portfolio show my experience as a software developer.**
+> ## Objectives
+- Build a Mobile First portafolio
+- Follow the best practices of adabtative content and Mobile First
+- Setup an skeleton 
+- Use flexBox
+- Follow GitHub flow.
+- Open pull requests in GitHub.
+
+
+## Built With
+- HTML
+- CSS
+
+## Getting Started
+- Please download clone the reporsitory
```

Comment:
- [ ] [OPTIONAL] Great job adding this as requested by the previous reviewer, however, I kindly suggest you use descriptive steps like below:

```
### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/LuisDavidRodriguez/Portfolio/archive/refs/heads/master.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/LuisDavidRodriguez/Portfolio.git
```

```
- Once the files are on your machine, open the _Portfolio_ folder in [Visual Studio Code](https://code.visualstudio.com/)

- With the files open in Visual Studio Code, press the **Go Live** button at the bottom of the window to launch the files with [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).
```
```